pkgs/tools/text/icdiff/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildPythonApplication rec {
```

Comment:
This should be also moved out of `pythonPackages`. It is not a python library therefor does not needs to be build against different versions of python.